app/classes/Transvision/ShowResults.php

```diff
@@ -58,6 +58,8 @@ public static function getSuggestionsResults($source_strings, $target_strings, $
         ];
         $flat_output = [];
 
+        $max_results = $max_results > 0 ? $max_results : 10;
```

Comment:
I think it would be good to add a comment explaining the issue and why we have to check that $max_results isn't equal to 0.

It would be good to also add a functional test and I think we don't have unit tests for this method.